datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -134,7 +138,12 @@ def __init__(self, instance, init_config, remapper=None, logger=None):
             config[field] = value
 
         # http://docs.python-requests.org/en/master/user/advanced/#timeouts
-        timeout = float(config['timeout'])
+        connect_timeout = read_timeout = float(config['timeout'])
+        if config['connect_timeout'] is not None:
+            connect_timeout = float(config['connect_timeout'])
+
+        if config['read_timeout'] is not None:
+            read_timeout = float(config['read_timeout'])
```

Comment:
Another way would be allowing a list in config. That would minimise the number of configs. What do you think ?

Example:

```yaml
    ## @param timeout - integer - optional - default: 5
    ## The timeout for connecting to the `server`
    #
    # timeout: [5, 10]
```